src/app/msw/index.ts

```diff
@@ -22,26 +22,25 @@ export const services = (_: Record<string, Token>): ServiceDefinition[] => {
       service: () => [(req: Request) => {
         const { pathname, href } = new URL(req.url)
         if (
-          !pathname.startsWith('/node_modules') &&
-         !pathname.startsWith('/src/assets/') &&
-         !passthroughExtension.test(href)
+          pathname.startsWith('/node_modules') ||
+          pathname.startsWith('/src/assets/') ||
+          passthroughExtension.test(href)
         ) {
-          return `Found an unhandled ${req.method} request to ${href}`
+          return true
         }
       }],
       labels: [
         $.mswPassthroughs,
       ],
-    },
-    ],
+    }],
     [$.msw, {
       service: (handlers: Handler[] = [], passthroughs: Passthrough[]) => {
         return setupWorker(...handlers).start({
           quiet: true,
-          onUnhandledRequest(req: Request) {
-            passthroughs.reduce<string[]>((prev, item) => prev.concat(item(req) ?? []), []).forEach(item => {
-              console.warn(item)
-            })
+          onUnhandledRequest(req: Request, print) {
+            if (!passthroughs.some((passthrough) => passthrough(req))) {
```

Comment:
In another PR there was a fix for `some` because it behaves different than expected in some cases. Is this the case here as well or are we safe here?

Reply:
Ah, I wasn't 100% sure so I just triple checked this. In this case it works as intended I think:


```
// when an MSW endpoint isn't found, some passthroughs defined
[shouldIPassThrough()].some((bool) => bool) // true, passthrough i.e. don't print a warning

// when an MSW endpoint isn't found, but there are no "passthroughs defined"
[].some((bool) => bool) // false don't passthrough i.e. make sure we print a warning
```